modules/order/src/Form/LineItemTypeForm.php

```diff
@@ -82,6 +82,8 @@ public function form(array $form, FormStateInterface $form_state) {
       '#default_value' => $lineItemType->getPurchasableEntityType(),
       '#options' => $purchasableEntityTypes,
       '#empty_value' => '',
+      '#disabled' => !$lineItemType->isNew() ? TRUE : FALSE,
```

Comment:
You can just do '#disabled' => !$lineItemType->isNew() since that already gives you a boolean.